_posts/2024-12-19-cosmicqc.md

```diff
@@ -0,0 +1,318 @@
+---
+title: "coSMicQC: A step toward quality control and improvement of morphology profiles"
+author: Jenna Tomkinson
+member: jenna-tomkinson
+tags:
+  - opinions
+  - process
+  - software
+---
+# Let's tour the single-cell morphological galaxy with coSMicQC!
+
+Hi everyone! 🤩
+
+Now that I have a bit of bandwidth, I wanted to take the time to come back to go over something that I am very excited and passionate about:
+
+✨ *Quality control of single-cell morphology profiles* ✨
+
+In this blog post, I will go over what is quality control for morphology profiles, what current work exists on this topic, the software (coSMicQC) that we have developed to help in this topic, and where we hope to go from here.
+
+## Contents
+
+1. 📋 [What is quality control for single-cell morphology profiles?](#defining-quality-control-of-single-cell-morphology-profiles)
+2. 📉 [What current exists for this topic?](#what-quality-control-methods-currently-exist)
+3. 🌌 [Intro to coSMicQC](#introduction-to-cosmicqc)
+4. 🎯 [Example of impact](#example-of-cosmicqc-impact)
+5. 🚀 [Where do we go from here?](#whats-next)
+
+## Defining quality control of single cell morphology profiles
+
+> **Quality Control (QC)**  
+> The process of validating and enforcing a specific standard across all products.
+
+> **Single-cell morphology profiles**  
+> Features (e.g., area, texture, intensity, etc.) extracted from images of cells. 
+> Traditionally, profiles are formatted in a data frame where each row is single-cell, and columns are either metadata or feature extracted from a specific segmented part of a cell (e.g., nucleus, whole cell, or cytoplasm). 
+
+Putting these two definitions together, single-cell quality control is the process of enforcing a standard quality based on single-cell morphology features, which we measure based on segmentation.
+
+### Background on image-based profiling
```

Comment:
It feels like this could be better in sequence if placed before the earlier section (to provide context for the definitions).

Reply:
Hmm I kinda like the definitions first to put the perspective of the next parts, but I can see both sides. I will keep for now, but thank you!